runtime/bcutil/ClassFileOracle.cpp

```diff
@@ -502,9 +502,8 @@ ClassFileOracle::walkAttributes()
 			break;
 		}
 		case CFR_ATTRIBUTE_PermittedSubclasses: {
-			/* PermittedSubclasses verification is for Java 15 preview only. Don't record the attribute for other class versions
-			 * since it may be corrupt. */
-			if ((59 == _classFile->majorVersion) && (0 < _classFile->minorVersion)) {
+			/* PermittedSubclasses verification is for Java version > 15 */
+			if (_classFile->majorVersion > 59) {
```

Comment:
```suggestion
			if ((_classFile->majorVersion > 59) 
			|| ((59 == _classFile->majorVersion) && (0 < _classFile->minorVersion))
			) {
```

Sorry, I spoke too quick.  We need to keep both the check for the preview version for the next release as that release includes jdk15.  Something like the above will cover us.

Needs to be applied both here and below

Reply:
Updated here and the same check in `cfreader.c`